nemo/collections/tts/torch/tts_tokenizers.py

```diff
@@ -525,3 +529,127 @@ def set_phone_prob(self, prob):
         finally:
             if hasattr(self.g2p, "phoneme_probability"):
                 self.g2p.phoneme_probability = self.phoneme_probability
+
+
+@experimental
+class PhonemizerTokenizer(IPATokenizer):
+    # fmt: off
+    # Common English punctiation. This is extended with language-specific punctuation during initialization.
+    # This can be overridden using "non_default_punct_list"
+    PUNCT_LIST = (
+        ',', '.', '!', '?', '-',
+        ':', ';', '/', '"', '(',
+        ')', '[', ']', '{', '}',
+    )
+    ESPEAK_SYMBOLS = ('ˈ', 'ˌ', "'", '-', 'ː')
+    # This is a common alphabet containing all IPA phonemes produced by eSpeak for English, Spanish, and German.
+    # This can be overridden using "non_default_chars"
+    CHARS = (
+        'a', 'b', 'd', 'e', 'f', 'h', 'i', 'j', 'k', 'l',
+        'm', 'n', 'o', 'p', 'r', 's', 't', 'u', 'w', 'x',
+        'ð', 'ŋ', 'ə', 'ɛ', 'ɟ', 'ɡ', 'ɣ', 'ɪ', 'ɲ', 'ɾ',
+        'ʃ', 'ʊ', 'ʎ', 'ʒ', 'ʝ', 'β', 'θ'
+    )
+    # fmt: on
+
+    def __init__(
+        self,
+        language,
+        phoneme_dict,
+        punct=True,
+        pad_with_space=False,
+        non_default_chars=None,
+        non_default_punct_list=None,
+        with_stress=True,
+        ignore_ambiguous_words=True,
+        heteronyms=None,
+        espeak_logging_level=ERROR,
+    ):
+        """IPA tokenizer using the Phonemizer library with an eSpeak backend
+
+        https://github.com/bootphon/phonemizer
+        http://espeak.sourceforge.net/
+
+        To use this class you will need to install eSpeak separately from NeMo:
+
+        apt-get update && apt-get install espeak-ng
```

Comment:
Will the import above cause an error if a user is not using this class but doesn't have espeak installed?

Reply:
No, it will only throw an error if they try to create an instance of this class (when EspeakBackend() is initialized)

Reply:
Gotcha, thanks.